.bazelrc

```diff
@@ -39,6 +39,12 @@ build --experimental_convenience_symlinks=ignore
 # frustrating when they fail and don't give any output. So, remove the limit.
 build --experimental_ui_max_stdouterr_bytes=-1
 
+# TODO(#11485) - enable bzlmod once it works
+common --noenable_bzlmod
+
+# TODO(#....) - apparently Bazel on macOS still needs work.
```

Comment:
I would reframe this to point to any grpc issue about not requiring rules you're not using. AFAIUI your case things are working as expected. Either you register the default toolchain first, you require users to have Xcode, or you remove apple_support, the later seems like the best solution for reducing your maintenance cost assuming grpc allowed you to only include what you need.

Reply:
Done.